.eslintrc.json

```diff
@@ -1,14 +0,0 @@
-{
-    "extends": ["prettier"],
-    "plugins": ["prettier"],
-    "ignorePatterns": ["*.min.js", "*bootstrap*", "/static_imports/*"],
-    "rules": {
-        "prettier/prettier": "error"
-    },
-    "env": {
-        "es6": true
-    },
```

Comment:
Is there an equivalent for this in `eslint.config.cjs`?

Reply:
default ecma version is "latest". is there a reason we want to use 6 specifically?

https://eslint.org/docs/latest/use/configure/language-options

Reply:
6 is a modern enough standard and it prevents us from trying to program toward a moving target. When you use `latest` as an option you can get in a scenario where builds break depending on when the `latest` version is changed.